tests/migration_test.py

```diff
@@ -0,0 +1,132 @@
+import os
+import pytest
+from unittest.mock import patch
+from migrations.migration import Migration
+from pytest import fixture
+
+
+class mock_response:
+    """Wrap data in response like object"""
+
+    def __init__(self, data, status_code=200):
+        self.data = data
+        self.status_code = status_code
+
+    def json(self):
+        return self.data
+
+    def raise_for_status(self):
+        if self.status_code == 200:
+            return
+        raise Exception("status code ain't 200")
+
+
+@fixture
+def migration_instance():
+    return Migration()
+
+
+@fixture
+def mock_get_previous_migration_id():
+    with patch.object(Migration, 'get_previous_migration_id') as mock:
+        yield mock
+
+
+def test_build_migration_sequence_empty():
+    # Mock the output of get_migration_files
+    with patch.object(Migration, 'get_migration_files', return_value=[]):
+        # Instantiate Migration class
+        migration_instance = Migration()
```

Comment:
puzzled to see `migration_instance` fixture, and calls to do the same?

Reply:
Some of the methods are easier to test from clean slate, instead of importing existing Migration. Can alter.